wbooks/src/app/components/ScreenWrapper/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { SafeAreaView, StatusBar } from 'react-native';
+
+import styles from './styles';
+
+interface Props {
+  children: React.ReactNode;
+}
+
+function ScreenWrapper({ children }: Props) {
```

Comment:
Delete tthis component, it's not necessary. Move the `StatusBar` to `src/app/index` and the `SafeAreaView` only in the `Library` screen

Reply:
Refactored.